### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by ArgusLi**
### Summary

Update Readme with new versioning

### Test Results

Not Applicable

### CODE REVIEW

**Review by ravjotbrar — APPROVED** [review]
Small nit to change the >=/<= to symbols to words like 1.2.0 and newer or 1.3.2 and older. Not a big deal, but does make readability better for the user.

**Comment by ArgusLi** [comment]
I think it's hard to explicitly say >= in words. If you look at the sentence in particular, "If the installed dbt-core has a version >=1.2 and <=1.3.2, the existing installation will be used and not upgraded to v1.3.2.", rephrasing the sentence would end up with something like "If the installed dbt-core has version between 1.2 and 1.3.2 inclusive, the existing installation will be used and not upgraded to v1.3.2".

In my opinion that is both more verbose and less readable, especially when the audience we are dealing with are data analysts.

File: README.md [hunk 1]
```diff
@@ -12,14 +12,14 @@ The `dbt-dremio` package contains all of the code enabling dbt to work with [Dre
 
 The dbt-dremio package supports both Dremio Cloud and Dremio Software (versions 22.0 and later).
 
-Installing the dbt-dremio package will install or update dbt-core to version 1.3.2.
+Installing the dbt-dremio package will install dbt-core to version 1.3.2 if there is no existing dbt-core installation or if the existing dbt-core installation is older than 1.2.0. If the installed dbt-core has a version >=1.2 and <=1.3.2, the existing installation will be used and not upgraded to v1.3.2.
```
Comment (jlarue26) [inline README.md:15]:
```suggestion
Version 1.3.2 of the dbt-dremio adapter is compatible with dbt-core versions 1.2.0 to 1.3.2.
```

File: README.md [hunk 2]
```diff
@@ -12,14 +12,14 @@ The `dbt-dremio` package contains all of the code enabling dbt to work with [Dre
...
 
 > Prior to version 1.1.0b, dbt-dremio was created and maintained by [Fabrice Etanchaud](https://github.com/fabrice-etanchaud) on [their GitHub repo](https://github.com/fabrice-etanchaud/dbt-dremio). Code for using Dremio REST APIs was originally authored by [Ryan Murray](https://github.com/rymurr). Contributors in this repo are credited for laying the groundwork and maintaining the adapter till version 1.0.6.5. The dbt-dremio adapter is maintained and distributed by Dremio starting with version 1.1.0b.
 
 ## Getting started
 
 -   [Install dbt](https://docs.getdbt.com/docs/installation)
-    -   Note that dbt-dremio requires dbt-core 1.3.2.
+    -   Note that dbt-dremio requires dbt-core >=1.2.0 and <=1.3.2.
```
Comment (jlarue26) [inline README.md:22]: I know that his content came from the other adapters' README, but it is a bit silly as it is not completely true.

Comment (jlarue26) [reply README.md:22]: Let's change line 22 to:

Install dbt-dremio (link to our documentation on dbt site)
- Version 1.3.2 of dbt-dremio requires dbt-core >= 1.2.0 and <=1.3.2. Installing dbt-dremio will automatically upgrade existing dbt-core versions earlier than 1.2.0 to 1.3.2, or install dbt-core v1.3.2 is no version of dbt-core is found.

Feel free to wordsmith the above.

**Review by jlarue26 — APPROVED** [review]
Looks good, thanks!

**Pull request merged**